epochtalk/epochtalk
# Recent threads

Recent threads section on front page

Testing Procedures:
Open the front page
Ensure the recent threads section exists
Make a post 
Ensure your post is shown on recent threads
Make a post is a board with a high priority
Ensure your post is shown on recent threads
Logout
Ensure your post is no longer shown on recent threads
